### PR TITLE
[Test-1] add test for FrameworkException,EnhancedServiceLoader,NamedThreadFactory,BlobUtil,NetUtil

### PR DESCRIPTION
**Pull request by xiaoheiAh**
### Ⅰ. Describe what this PR did

add more ut for common
### Ⅱ. Does this pull request fix one issue?

nope

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
pr just test

### Ⅳ. Describe how to verify it
run ut first

### CODE REVIEW

File: common/src/test/java/com/alibaba/fescar/common/exception/FrameworkExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/*
+ *
+ *  *  Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *  *
+ *  *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  *  you may not use this file except in compliance with the License.
+ *  *  You may obtain a copy of the License at
```
Comment (slievrly) [inline common/src/test/java/com/alibaba/fescar/common/exception/FrameworkExceptionTest.java]: copyright is not right， double *？

File: common/src/test/java/com/alibaba/fescar/common/util/BlobUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        assertThat(BlobUtils.string2blob("123abca")).isEqualTo(new SerialBlob("123abca".getBytes()));
+    }
+
+    @Test
+    public void testBlob2string() throws SQLException {
+        assertThat(BlobUtils.blob2string(new SerialBlob("123abckdle".getBytes()))).isEqualTo("123abckdle");
+    }
+
+    @Test
+    public void testInputStream2String() {
```
Comment (slievrly) [inline common/src/test/java/com/alibaba/fescar/common/util/BlobUtilsTest.java]: UTF8?

File: common/src/test/java/com/alibaba/fescar/common/util/NetUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+import org.junit.Test;
+
+/**
+ * @author melon.zhao
+ * @since 2019/2/26
+ */
+public class NetUtilTest {
+
+    private InetSocketAddress ipv4 = new InetSocketAddress(Inet4Address.getLocalHost().getHostName(), 3902);
+    private InetSocketAddress ipv6 = new InetSocketAddress(Inet4Address.getLocalHost().getHostName(), 3904);
```
Comment (slievrly) [inline common/src/test/java/com/alibaba/fescar/common/util/NetUtilTest.java]: ipv6？

File: common/src/test/java/com/alibaba/fescar/common/loader/ChineseHello.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * @author melon.zhao
+ * @since 2019/2/26
+ */
+@LoadLevel(name = "ChineseHello",order = Integer.MIN_VALUE)
+public class ChineseHello implements Hello{
+
+    @Override
+    public String say() {
+        return "你好!";
```
Comment (slievrly) [inline common/src/test/java/com/alibaba/fescar/common/loader/ChineseHello.java]: Change English  “ ChineseHello” ？

File: common/src/test/java/com/alibaba/fescar/common/util/BlobUtilsTest.java (hunk 2 above)
Comment (xiaoheiAh) [reply common/src/test/java/com/alibaba/fescar/common/util/BlobUtilsTest.java]: if i need define a constant string with utf-8?

File: common/src/test/java/com/alibaba/fescar/common/util/NetUtilTest.java (hunk 3 above)
Comment (xiaoheiAh) [reply common/src/test/java/com/alibaba/fescar/common/util/NetUtilTest.java]: error code,will fix

**Pull request closed without merging**